spec/unit/provider/powershell_script_spec.rb

```diff
@@ -54,32 +32,12 @@ def set_user_defined_flag
 
   context "when setting interpreter flags" do
     before(:each) do
-      allow(provider).to receive(:is_forced_32bit).and_return(false)
-      os_info_double = double("os_info")
-      allow(provider.run_context.node["kernel"]).to receive(:[]).with("os_info").and_return(os_info_double)
-      allow(os_info_double).to receive(:[]).with("system_directory").and_return("C:\\Windows\\system32")
+      allow(provider).to receive(:basepath).and_return("C:\\Windows\\system32")
     end
 
     it "sets the -File flag as the last flag" do
       flags = provider.command.split(" ").keep_if { |flag| flag =~ /^-/ }
       expect(flags.pop).to eq("-File")
     end
-
-    let(:execution_policy_flag) do
-      provider_flags = provider.flags.split(" ")
-      # Last occurance of "executionpolicy"
-      execution_policy_index = provider_flags.map(&:downcase).rindex("-executionpolicy")
-
-      execution_policy_index ? provider_flags[execution_policy_index + 1] : nil
-    end
-
-    it "sets default -ExecutionPolicy flag to 'Bypass'" do
-      expect(execution_policy_flag).to eq("Bypass")
-    end
-
-    it "sets user defined -ExecutionPolicy flag to 'RemoteSigned'" do
-      set_user_defined_flag
-      expect(execution_policy_flag).to eq("RemoteSigned")
-    end
```

Comment:
This was testing the behavior of a property on the resource, so I moved the tests for that there.